packages/common/src/types.ts

```diff
@@ -47,7 +47,7 @@ export interface GenesisBlockConfig {
 export interface HardforkConfig {
   name: Hardfork | string
   block: number | null
-  td?: number
+  ttd?: bigint | string
```

Comment:
Do we really need a bigint type here? If I see correctly `HardforkConfig` is solely(as the other *Config types) for data coming from JSON and JSON doesn't support bigint? This is then not consistent with the other number related types, e.g. in `GenesisBlockConfig` we do `number` for numbers and `string` for values which we later use as BigInts.